.gitignore

```diff
@@ -41,3 +41,4 @@ nb-configuration.xml
 .envrc
 .jekyll-cache
 .mvn/.gradle-enterprise
+.quarkus
```

Comment:
What was this for?

Reply:
When I ran the test, a `.quarkus/cli/quarkus-catalog.json` is created in the root, see https://github.com/quarkusio/quarkus/commit/558aadf2392bcbe34645b897f50b86d75d2d5ae5